cmd/geth/chaincmd.go

```diff
@@ -206,6 +221,64 @@ func getIsQuorum(file io.Reader) bool {
 	return altGenesis.Config.IsQuorum == nil || *altGenesis.Config.IsQuorum
 }
 
+// updateTransitions will update genesis block with the new transitions data
+func updateTransitions(ctx *cli.Context) error {
+	// Open and initialise both full and light databases
```

Comment:
recommend moving this database opening close to the forloop for updating the DBs

Reply:
amended